Fix KubePersistentVolumeFillingUp duplicates (#22)
projectsyn/component-rancher-monitoring: docs/modules/ROOT/pages/references/parameters.adoc

    runbook_url: https://www.google.com/?q=Watchdog
----

== `alerts.sharedStorageClass`

[horizontal]
type:: string
default:: ``

A regular expression that matches the shared storage classes in this cluster.

> With that we mean storage classes which PVs share the same underling volume, which causes them to fill up at the same rate.

Reviewer: We tend not to use phrasing that you'd find in a research paper in our reference documentation.

```suggestion
A shared storage class is a storage class for which PVs share the same underlying volume, which causes them to fill up at the same rate.
```

> Matching storage classes will only generate a single alert for all PVs.

Reviewer: ```suggestion
The component configures the alert rules to ensure that only a single alert is produced for storage classes matching this regex.
```

> Make sure this only matches storage classes that have this property.

Reviewer: ```suggestion
Users must ensure that the regex only matches storage classes which share a single backing volume.
```

> Otherwise alerts will be lost.

Reviewer: ```suggestion
Otherwise volume utilization alerts will be lost.
```
